src/Apps/Artists/Routes/ArtistsByLetter.tsx

```diff
@@ -88,9 +88,17 @@ export const ArtistsByLetter: React.FC<
     })
   }
 
+  const title = `Artists Starting with ${params.letter.toUpperCase()} | Modern and Contemporary Artists`
+  const description = `Research and discover artists starting with ${params.letter.toUpperCase()} on Artsy. Find works for sale, biographies, CVs, and auction results.`
+
   return (
     <>
-      <ArtistsByLetterMeta />
```

Comment:
I deleted ArtistsByLetterMeta because it was becoming a thin wrapper for MetaTags. (If anything we may want a PaginatedMetaTags wrapper if that component becomes too complex.)